GP350.py

```diff
@@ -0,0 +1,54 @@
+from pymeasure.instruments import Instrument, Channel
+import numpy as np
+
+
+class GP_PressureChannel(Channel):
+    """
+    Channel of the GP350UHV vacuum sensor for pressure measurement.
+
+    Each channel corresponds to a filament or display output of the sensor.
+    """
+
+    pressure = Instrument.measurement(
+        "#RD{ch}",
+        """Measure the current pressure in mbar (float).""",
+        preprocess_reply=lambda reply: reply.lstrip("* ").strip(),
+        get_process=lambda r: np.nan if float(r) == 9.90e9 else float(r),
```

Comment:
There is also a `float("nan")` for nans without requiring numpy.

Is it nan or is it infinty? You could then use also `float("inf")` to indicate an overrange.